lib/platform/common.ts

```diff
@@ -116,6 +116,7 @@ export interface Pr {
   state: string;
   targetBranch?: string;
   title: string;
+  draft?: boolean;
```

Comment:
```suggestion
```

Unused, so remove

Reply:
I am using it in tests and could be used for later. Do you still want me to remove it?

Reply:
Removed this in latest commit fe70f80